internal/handlers/commonparams/extract_params.go

```diff
@@ -37,10 +40,30 @@ import (
 //   - `opt` - field is optional;
 //   - `non-default` - field is unimplemented for non-default values;
 //   - `unimplemented` - field is not implemented yet;
-//   - `ignored` - field is ignored.
+//   - `ignored` - field is ignored;
+//   - `numericBool` - field is of type `bool` but can be parsed from numeric types;
+//   - `strict` - fields of numeric types are parsed strictly;
+//   - `positive` - fields of numeric types must be positive;
+//   - `numericAsBool` - fields of numeric types that must have as values only `0` or `1`.
```

Comment:
Some of those tags are hard to understand. What is the default value for `opt`? Why `non-default` is not implemented? (I guess it should be `unimplemented-non-default`, but then what is the default value?) `strict` can't be defined as "strict" (you can't define X in terms of X itself). Etc

Reply:
I rewrote the descriptions to explain what those tags should be used for.

Reply:
> `opt` - field is optional, the field value would not be set if it's not present in the document;

Ok, so if the field is required, will it be set if it is not in the document? That's pretty much the opposite of being required, no?

Based on that documentation, I don't know how to use that function